server/embed/config.go

```diff
@@ -517,6 +513,10 @@ type configJSON struct {
 	PeerSecurityJSON   securityConfig `json:"peer-transport-security"`
 
 	ServerFeatureGatesJSON string `json:"feature-gates"`
+	// ExperimentalCompactionBatchLimit Sets the maximum revisions deleted in each compaction batch.
+	// Deprecated in v3.6 and will be decommissioned in v3.7.
+	// TODO: Delete in v3.7
+	ExperimentalCompactionBatchLimit int `json:"experimental-compaction-batch-limit"`
```

Comment:
You just moved them from line 391-395 to here?

Reply:
I moved it from `Config` to `configJSON` to support JSON config. Let me update PR name